tests/src/test/scala/whisk/core/entity/test/ExecManifestTests.scala

```diff
@@ -170,4 +176,22 @@ class ExecManifestTests
                     image.localImageName("r", "p", Some("tag")) shouldBe s"r/p/$name:tag"
             }
     }
+
+    it should "throw an error when configured manifest is invalid" in {
+        val config_manifest = "{\"nodejs\":[{\"kind\":\"nodejs\",\"image\":{\"name\":\"nodejsaction\"},\"deprecated\":true},{\"kind\":\"nodejs:6\",\"default\":true,\"image\":{\"name\":\"nodejs6action\"}}]}"
```

Comment:
Thanks for the suggestion. The reason I went with string initially is so that I can create either an invalid JSON, or an invalid manifest even if that JSON is valid, the goal being to simulate errors in config.  Indeed a `"""` would have been 👍